modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/ConfigInitializerContextListener.java

```diff
@@ -116,6 +120,7 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
         properties.setNotBeforeDays(signedCertNotBeforeDate);
         properties.setNotAfterDays(signedCertNotAfterDate);
         properties.setAuthPolicy(authPolicy);
+        properties.setDomain(domain);
         servletContext.setAttribute(Constants.WINDOWS_PLUGIN_PROPERTIES, properties);
 
         File wapProvisioningFile = new File(getClass().getClassLoader().getResource(
```

Comment:
Class name "ConfigInitializerContextListener" as it sounds, seems like a combination of two classes. naming seems inappropriate.